doc/tools/coqrst/coqdomain.py

```diff
@@ -1180,8 +1180,7 @@ def ci_sort(entry):
             if type == 'term':
                 entries = content[itemname[0].lower()]
                 entries.append([itemname, 0, docname, anchor, '', '', ''])
-        content = sorted(content.items())
-        return content, False
+        return content.items(), False
```

Comment:
What is this about?

Reply:
IIRC it's a superfluous sort.  Removing it makes no difference to the order of glossary entries, which is determined by the "for ... sorted" statement above.